src/app/home/home.component.html

```diff
@@ -123,17 +123,25 @@ <h1 i18n>Planet Learning</h1>
       </ul>
       <!--Bottom part-->
       <ul class="bottom-nav">
+        <li>
+          <a mat-button planetFeedback i18n-title title="Feedback">
+            <mat-icon>help_outline</mat-icon>
+            <label i18n>Feedback</label>
+          </a>
+        </li>
         <li>
           <a mat-button routerLink="/surveys" i18n-title title="Surveys">
             <mat-icon svgIcon="surveys"></mat-icon>
             <label i18n>Surveys</label>
           </a>
         </li>
         <li>
```

Comment:
li is not required to create if condition does not match
Also you can just add auth directive to li no need for ng-container

Reply:
oh it also needs to check if planet is **online**

Reply:
ok. please check the changes now.. thanx in advance @lmmrssa